tensorflow/tools/compatibility/ipynb.py

```diff
@@ -62,8 +62,37 @@ def process_file(in_filename, out_filename, upgrader):
   return files_processed, report_text, errors
 
 
+def skip_magic(code_line, magic_list):
+  """
+  Checks if the cell has magic, that is not python-based.
+
+   >>> skip_magic('!ls -laF', ['%', '!', '?'])
+  True
```

Comment:
Can you add an args, returns section, and put the brief description on the same line as the opening """? (also below)